package.json

```diff
@@ -34,7 +34,7 @@
     "extract-text-webpack-plugin": "3.0.2",
     "file-loader": "1.1.5",
     "flux-standard-action": "^2.0.1",
-    "frontend-common-components": "github:entando/frontend-common-components#master",
+    "frontend-common-components": "github:entando/frontend-common-components#v5.0.0",
```

Comment:
it should be pointing to version 5.0.1

Reply:
I will omit changes to package.json from the PR, they are not relevant to what is in the update